docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[Kubernetes-dashboard]]
+= The Kubernetes dashboard: Linux session data from your Kubernetes clusters
+
+This page explains how to deploy a https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/[daemonset] to collect session data from your Kubernetes clusters using the {endpoint-cloud-sec} integration, and how to view that data in Kibana’s Kubernetes dashboard.
+
+**Prerequisites**: This feature requires Elastic Stack version 8.4 or higher. Your Elastic deployment must have the {endpoint-cloud-sec} integration {security-guide}/install-endpoint.html[enabled], and you need an active {fleet-guide}/fleet-overview.html[{fleet} Server].
+
+**Support Matrix**: This feature is currently available on GKE and EKS using Linux hosts and Kubernetes versions that match the following specifications:
+|=====================
+| | **Kubernetes versions** | **Container runtime** | **Nose OSes**
+|**EKS**| 1.18; 1.19; 1.20; 1.21| Docker | Amazon Linux 2, Bottlerocket OS
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+You first need to download the daemonset manifest .yaml, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor. After that, data will start to appear in the Kubernetes dashboard.
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+1. Download the daemonset manifest http://github.com/elastic/placeholder[here] for GKE or http://github.com/elastic/placeholder[here] for EKS.
+2. Fill in the manifest's `FLEET_URL` field with your {fleet} server's `Host URL`. To find it, go to **Kibana > Management > {fleet} > Settings**. For more information, refer to {fleet-guide}/fleet-settings.html[Fleet UI settings].
+3. Fill in the manifest's `FLEET_ENROLLMENT_TOKEN` field with a Fleet enrollment token. To find one, go to **Kibana > Management > {fleet} > Enrollment tokens**. For more information, refer to {fleet-guide}/fleet-enrollment-tokens.html[Fleet enrollment tokens].
+
+IMPORTANT: The Elastic Agent policy you use must include the {endpoint-cloud-sec} integration, and must have `Include session data` set to `true`. To modify this setting, go to **Security > Manage > Policies**, select your policy, and find `Include session data` near the bottom of the `Policy settings` tab:
+
+image::session-data-config.png[the "Enable session data" setting on the Policy settings page]
```

Comment:
If enabling session data is a requirement, then it should probably be in prerequisites or a numbered step somewhere earlier. You could still use an "Important" admonition to reinforce this requirement, but I wouldn't wait until this late point to mention it for the first time.

I'm not sure we need a screenshot for this, considering the maintenance cost. The instructions are clear enough, and we also opted not to use a screenshot for the main [Session View docs](https://www.elastic.co/guide/en/security/master/session-view.html#enable-session-view). But if an image is necessary, in addition to adding a border, crop out the side nav and top status bar, which will surely change in the future.